app/controllers/api/v1/merchants/revenues_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::Merchants::RevenuesController < ApplicationController
```

Comment:
Debating renaming Merchants::RevenuesController to Merchants::MostRevenuesController, as there is a RevenuesController not nested under Merchants already